### ISSUE TITLE
Search auto-suggestions and user results are very different

### ISSUE THREAD

**Issue by alimony** [bluesky-social/social-app#715]
When I search for a user in the search field, I get auto-suggested results inline. These seem to map well to the search term, see for example a search for "abc":

<img width="323" alt="Screenshot 2023-05-17 at 18 07 00" src="https://github.com/bluesky-social/social-app/assets/331091/ea771729-2478-4665-8993-121c3dc56f86">

However, if I run the same search by pressing enter to get final results (posts and users) and then switch to the users results, they are completely different and often quite wrong. Here are the results for the same "abc" search term:

<img width="623" alt="Screenshot 2023-05-17 at 18 07 26" src="https://github.com/bluesky-social/social-app/assets/331091/632ed991-5c12-44d4-977f-e73a582d8773">

I'm guessing what happens is that a search always searches posts, and the "Users" results are based on that somehow, rather than direct results from the search term.

**Details**

 - Platform: Web
 - Platform version: Firefox 113.0.1
 - App version: Not sure (web still says version "TODO")

**Comment by pfrazee** [comment]
Yeah they're two different systems. Both need a lot of improvements but just haven't been top priority.

**Comment by mschwendener** [comment]
After I click for searching, and then switch to the tab 'People', could you just show the accounts first which have _abc_ in their name or their handle, and only afterwards the ones that have _abc_ in the description? 

<img width="585" alt="screenshot_4927" src="https://github.com/bluesky-social/social-app/assets/4497705/e93ad80c-f2a4-4993-a532-302cd2fc462d">
